<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="c0016393" failures="31" tests="124" name="prerate.prerateTestNGSlow" time="3616.807" errors="3" timestamp="2020-12-15T23:03:21 UTC" skipped="0">
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.953"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.807">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.851">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.001"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="25.909">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="24.828"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.115"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="26.477"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.094"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="18.882"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.522"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="60.926">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="61.648">
    <failure message="Sum of Charge Details does not equal Payment Total Amount" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Sum of Charge Details does not equal Payment Total Amount
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1248)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.345">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="44.197"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="30.836"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="53.964">
    <failure message="Technical Error Occurred in Edits; Try again ,7368" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,7368
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.375"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="62.359">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.526"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.749"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.382"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.469"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.033"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.593"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="30.864"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.084"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.926"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.614"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.098"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="26.330">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.270"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.255"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.107"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.179"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.297"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.292"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="18.768"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="43.575"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="18.617"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.386">
    <failure message="Technical Error Occurred in Edits; Try again ,6966,7169" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,6966,7169
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.131">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.280"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.599"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="53.074">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="45.848"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.884"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="53.403">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.532"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.776"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.757"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="45.003"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.041"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="56.844">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.098"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.229"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.729"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="25.802">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="27.730"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.752"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.119"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.328">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.950"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="52.079">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.408">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.894"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.803"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.525"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.729"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.814"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod2" time="0.000">
    <error message="
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod2() returned a null value" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod2() returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:147)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod2 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.263"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.166"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.949"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="25.621">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="42.040"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="53.752">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.462"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.588"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="18.649"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="41.802"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.848"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.684"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.297"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.215"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="51.108">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.754"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="26.347">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="48.564"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod3" time="0.000">
    <error message="
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod3() returned a null value" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod3() returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:147)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod3 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="43.791"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="54.975">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="25.109"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.521"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="25.554"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.214"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.806"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod4" time="0.000">
    <error message="
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod4() returned a null value" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod4() returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:147)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- testMethod4 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.264"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="28.994">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="29.540">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="18.697"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.002"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.052"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.471"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.539"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.928"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="21.104"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="27.294"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="19.480"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="53.567">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="18.903"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="47.005"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.825"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="54.897">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="28.065">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="43.328"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="18.993"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="25.895">
    <failure message="Failed selecting dropdown menu..." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed selecting dropdown menu...
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1012)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="53.743">
    <failure message="Technical Error Occurred in Edits; Try again 0006" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again 0006
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="20.916"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="54.093">
    <failure message="Technical Error Occurred in Edits; Try again ,3310" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
at org.testng.Assert.fail(Assert.java:99)
at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="22.173"/>
  <system-out/>
  <testcase classname="prerate.prerateTestNGSlow" name="testMethod1" time="23.296"/>
  <system-out/>
</testsuite> <!-- prerate.prerateTestNGSlow -->
